flutter/news/.gitignore

```diff
@@ -11,6 +11,7 @@
 .history
 .svn/
 *.mocks.dart
+*.g.dart
```

Comment:
Parts generated by `retrofit` or `json_serialization` have the extension `.g.dart`, ignoring them